broker/src/main/java/org/apache/rocketmq/broker/BrokerController.java

```diff
@@ -388,6 +388,20 @@ public void run() {
                 }
             }, 1000 * 10, 1000 * 60, TimeUnit.MILLISECONDS);
 
+            if (this.brokerConfig.isWatchNamesrvAddr()) {
+                this.scheduledExecutorService.scheduleAtFixedRate(new Runnable() {
+
+                    @Override
+                    public void run() {
+                        try {
+                            BrokerController.this.brokerOuterAPI.updateNameServerAddressList(brokerConfig.getNamesrvAddr());
+                        } catch (Throwable e) {
+                            log.error("ScheduledTask updateNameServerAddressList exception", e);
+                        }
+                    }
+                }, 1, 1, TimeUnit.MINUTES);
```

Comment:
It would be better to keep consistent with the scheduled tasks in org.apache.rocketmq.broker.BrokerController#initialize(fetchNameServerAddr task)

Reply:
Frequency changed to keep consistent.